wbooks/src/app/screens/BookList/index.tsx

```diff
@@ -1,18 +1,32 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { SafeAreaView, FlatList, ListRenderItem } from 'react-native';
-import { BOOKS_MOCK } from '@constants/mockBooks';
+import { useDispatch, useSelector } from 'react-redux';
 import { Book } from '@interfaces/Book';
+import actionCreators from '@redux/book/actions';
 
 import BookItem from './components/BookItem';
 import styles from './styles';
 
+interface Props {
+  bookReducer: {
+    books: Book[];
+  };
+}
```

Comment:
Why this? You must define an AppState interface (in another file) for this:
```
interface AppState {
 book: BookState;
}
```
And define BookState with the book reducer state interface